app/api/v1/endpoints/auth.py

```
@@ -1,40 +1,590 @@
+        raise HTTPException(
+            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            detail="Sign out failed"
+        )
+
+
+@router.post("/refresh", response_model=AuthResponse)
+@limiter.limit("60/minute")
+async def refresh_token(
+    request: Request,
+    refresh_token_value: str,
+    auth_client: SupabaseAuthClient = Depends(get_auth_client)
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)